packages/core/ios/Tests/RNDdSdkConfigurationTests.swift

```diff
@@ -0,0 +1,95 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2021 Datadog, Inc.
```

Comment:
very minor: I notice that for some reason the ios copyright notice for new files shows 2019-2021 -> shouldn't it be until the present?